CHANGELOG.rst

```diff
@@ -7,9 +7,10 @@ in development
 * Report a more user-friendly error if an action-chain task references an invalid or inexistent
   action. Also treat invalid / inexistent action as a top-level action-chain error. (improvement)
 * Report a more user-friendly error if an action-chain definition contains an invalid type.
-  (improvements)
+  (improvement)
 * Don't allow ``run-remote-script`` actions without an ``entry_point`` attribute - throw an
   exception when running an action. (improvement)
+* Rename all st2 processes to be prefixed by st2. (sensor_container is now st2sensorcontainer, rules_engine is now st2rulesengine, actionrunner is now st2actionrunner) (improvement)
```

Comment:
We should probably also have a separate "Upgrade notes" section in the documentation where we describe backward incompatible / breaking changes like this one.

Something like this - https://libcloud.readthedocs.org/en/latest/upgrade_notes.html